ultron/src/main/java/com/atiurin/ultron/core/espresso/recyclerview/UltronRecyclerViewItem.kt

```diff
@@ -77,6 +78,9 @@ open class UltronRecyclerViewItem {
     fun withAssertion(name: String = "", isListened: Boolean = false, block: () -> Unit) =
         getMatcher().withAssertion(DefaultOperationAssertion(name, block.setListenersState(isListened)))
 
+    //root view searching
```

Comment:
Pls, add a javadoc with description of the method purpose. User should understand the usecase

Reply:
https://github.com/open-tool/ultron/wiki/Avoiding-nontrivial-element-lookup-exceptions